doc/source/ray-contribute/development.rst

```diff
@@ -74,39 +133,55 @@ For RHELv8 (Redhat EL 8.0-64 Minimal), run the following commands:
   sudo yum groupinstall 'Development Tools'
   sudo yum install psmisc
 
-Install bazel manually from link: https://docs.bazel.build/versions/main/install-redhat.html 
-
+In RedHat, install bazel manually from this link: https://docs.bazel.build/versions/main/install-redhat.html
```

Comment:
Shall we try to keep Bazel uppercase consistently?

Reply:
I like that! I did the same for Brew too.